dcat/index.html

```diff
@@ -5971,6 +5971,7 @@ <h2>Acknowledgments</h2>
   
   Addison Phillips, 
   Alex Nelson,
+  Andreas Geißner,
   Andreas Kuckartz, 
   Anna Odgaard Ingram, 
   <!-- Armando Stellato, --> 
```

Comment:
```suggestion
Annette Greiner
  <!-- Armando Stellato, --> 
```

Reply:
Annette's thanks are already included in the previous paragraph. This one is about contributors who are not members of the WG.